.changeset/kind-icons-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/compiler': minor
```

Comment:
I think `minor` is "nicer," but... this is technically a `major` breaking change, right? Version numbers are free!

Reply:
Discussed on Discord and we decided to go with `minor`.